Makefile

```
@@ -1,5 +1,5 @@
 .PHONY: peers dev seed test test-race cover cover-html lint fmt generate outdated db-up db-down pot catalogs translations \
-	translations-retire \
+	translations-push translations-retire \
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)